modules/rostests/apitests/ntdll/probelib.c

```diff
@@ -31,21 +31,31 @@ QuerySetProcessValidator(
                     break;
                 }
 
+                /*
+                 * NOTE: Buffer validation checks are relaxed on Windows Server 2003 SP2,
+                 * that is, the system only checks if the pointer is properly aligned in
+                 * NtQueryInformationProcess function and friends but it doesn't check if the
+                 * buffer is valid (that is up for each information class to care about that, whatever
+                 * that is). On ReactOS however we're way aggressive regarding this.
+                 */
                 case ProcessHandleTracing:
                 {
+#ifdef __REACTOS__
+                    SpecialStatus = STATUS_ACCESS_VIOLATION;
+#else
                     SpecialStatus = STATUS_INVALID_PARAMETER;
+#endif
```

Comment:
Are you changing expected test results based on whether we're compiling this in Ros tree or not ?

Reply:
looks like it does.

Reply:
I was experimenting some stuff, apologies for having amended that into commit in advance.